source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/FileFormat.java

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright the State of the Netherlands
+ * Crown copyright
```

Comment:
Not sure if we have a license check here, but is this double copyright statement OK?